lib/CurrencyExchangeRateFields/CurrencyExchangeRateFields.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState, useCallback, useEffect } from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Field } from 'react-final-form';
+
+import {
+  Checkbox,
+  Col,
+  Row,
+} from '@folio/stripes/components';
+
+import { TextField } from '../Fields';
+import { TooltippedControl } from '../TooltippedControl';
+import { validateRequired } from '../utils';
+import { FieldCurrency } from '../Currency';
+import CurrentExchangeRate from './CurrentExchangeRate';
+
+const CurrencyExchangeRateFields = ({
+  change,
+  currencyFieldName,
+  currencyRequired,
+  exchangeRate,
+  exchangeRateFieldName,
+  filledCurrency,
```

Comment:
how filledCurrency different from initialCurrency?

Reply:
`initialCurrency` is from `initialValues`
`filledCurrency` is current currency value set by user.

Reply:
`const { values } = useFormState();` and `values?.[fieldName]` gives you the same